imports/api/Users/index.js

```diff
@@ -2,9 +2,15 @@ import { SimpleSchema } from 'meteor/aldeed:simple-schema'
 import { Meteor } from 'meteor/meteor'
 import { Mongo } from 'meteor/mongo'
 
+const ProfileAvatarSchema = new SimpleSchema({
+  original: { type: String },
+  x250: { type: String, optional: true },
```

Comment:
Its just a field name where we would store the url to a resized avatar image at 250x250 pixels.